src/components/Carousel/CarouselDots.tsx

```diff
@@ -103,13 +103,13 @@ const CarouselDots = forwardRef<HTMLDivElement, CarouselDotsProps>(
       >
         <div className={`${baseClassName}-pagination-container`}>
           <div className={`${baseClassName}-pagination-container-inner`}>
-            {visibleDots.map((snapPoint, index) => {
+            {visibleDots.map((_, index) => {
               const actualIndex = scrollSnaps.indexOf(visibleDots[index]);
               const isSelected = selectedIndex === actualIndex;
 
               return (
                 <button
-                  key={`${componentId}-dot-${snapPoint}`}
+                  key={`${componentId}-dot-${index}`}
```

Comment:
added this back for now, otherwise it's throwing a key warning
`snapPoint` array gets initialized with all zero's, so it has identical keys on first render

Reply:
i can make it required, or default it to the empty string

was trying to make it match the img api